src/display/annotation_layer.js

```diff
@@ -1700,7 +1705,7 @@ class PopupElement {
       (!this.contentsObj?.str || this.contentsObj.str === this.richText.str)
     ) {
       XfaLayer.render({
-        xfa: this.richText.html,
+        xfaHtml: this.richText.html,
```

Comment:
> Is the reason for changing this to avoid confusion w.r.t. the parameter's meaning,

Exactly, since the fact that `XfaLayerBuilder` used `xfaHtml` but the `XfaLayer` used `xfa` *really* confused me initially when adding the missing `XfaLayerParameters` typedef.